x-pack/plugins/banners/server/ui_settings.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { i18n } from '@kbn/i18n';
+import { UiSettingsServiceSetup } from 'src/core/server';
+import { isHexColor } from './utils';
+
+// Note: unused for now, will be used for v2 (see https://github.com/elastic/kibana/issues/17298)
+
+export const registerSettings = (uiSettings: UiSettingsServiceSetup) => {
+  uiSettings.register({
+    'banner:placement': {
+      name: i18n.translate('xpack.banners.settings.placement.title', {
```

Comment:
I kept them in the PR, even if those will not be used until stage 2. I can revert these files if we really want to.

Reply:
I'd vote for removing them for now until we need them in stage 2. since we might want to change some details anyways.